### DISCUSSION TITLE
纯小白 再次求助🙏😭🙏

### DISCUSSION THREAD

**Discussion by MstZzz** [SAGIRI-kawaii/sagiri-bot#418] [Q&A] [answered]
请问运行 python main.py的时候显示如下报错该怎么处理？

ModuleNotFoundError: No module named 'loguru'

**Comment by SAGIRI-kawaii** [comment, marked as answer]
- 如果你使用`conda`，先运行`poetry install`，再运行`python main.py`
- 如果你只使用 `poetry`，先运行`poetry install`，再运行`poetry run python main.py`

**Reply by MstZzz** [reply]
![image](https://user-images.githubusercontent.com/125751048/225938675-404baa1b-3410-443d-9d7d-08495cdaa3d2.png)
运行了poetry run python main.py 结果是这样 怎么回事啊大佬

**Reply by SAGIRI-kawaii** [reply]
查看配置文件的[文档](https://sagiri-kawaii.github.io/sagiri-bot/configuration/)

**Reply by MstZzz** [reply]
看不太明白啊大佬😭😭
请您细🔒一下好不好🙏🙏🙏

**Reply by SAGIRI-kawaii** [reply]
后面不都是中文吗。。。哪里看不懂可以问
